Mailbox/Mailbox.cs

```diff
@@ -1,4 +1,40 @@
-﻿namespace Mailbox
+﻿using System;
+
+namespace Mailbox
 {
+    public class Mailbox
+    {
+        public Size Size { get; set; }
+        public ValueTuple<int, int> Location { get; set; }
+        public Person Owner { get; set; }
+
+        public Mailbox(Size size, ValueTuple<int, int> location, Person owner)
```

Comment:
This works but value tuples have a simpler syntax:
```suggestion
        public Mailbox(Size size, (int, int) location, Person owner)
```